File: Go/exercicio8.go
package main

import "fmt"

type Empregado struct {
    Nome   string
    Cargo  string
    Salario float64
}

type Empresa struct {
    Empregados []Empregado
}

func (e *Empresa) AdicionarEmpregado(empregado Empregado) {
    e.Empregados = append(e.Empregados, empregado)
}

func main() {
    empresa := Empresa{}
    empresa.AdicionarEmpregado(Empregado{"João", "Gerente", 5000})
    empresa.AdicionarEmpregado(Empregado{"Ana", "Desenvolvedora", 4000})

    for _, empregado := range empresa.Empregados {
        fmt.Printf("%s, %s, R$%.2f\n", empregado.Nome, empregado.Cargo, empregado.Salario)
    }
}

File: Python/exercicio8.py
class Empregado:
    def __init__(self, nome, cargo, salario):
        self.nome = nome
        self.cargo = cargo
        self.salario = salario

class Empresa:
    def __init__(self):
        self.empregados = []

    def adicionar_empregado(self, empregado):
        self.empregados.append(empregado)

empresa = Empresa()
empresa.adicionar_empregado(Empregado("João", "Gerente", 5000))
empresa.adicionar_empregado(Empregado("Ana", "Desenvolvedora", 4000))

for empregado in empresa.empregados:
    print(f"{empregado.nome}, {empregado.cargo}, R${empregado.salario}")